data/countries/de/ni/schaumburg/data.yaml

```
@@ -0,0 +1,33 @@
+---
+type: REGIONAL_CHAPTER
+level: DE:KREISVERBAND
+country: DE
+state: Niedersachsen
+district: Schaumburg
+urls:
+  - type: WEBSITE
+    url: http://www.gruene-schaumburg.de/
+---
+type: REGIONAL_CHAPTER
+level: DE:ORTSVERBAND
+country: DE
+state: Niedersachsen
+district: Schaumburg
+city: Bückeburg
+urls:
+  - type: WEBSITE
+    url: http://www.gruene-bueckeburg.de/
```

```suggestion
    url: https://gruene-schaumburg.de/stadtverband-bueckeburg/
```

Die URL hat sich geändert.